js/app.js

```diff
@@ -37,56 +54,49 @@ Question.prototype = {
     }
 };
 
-// Details for each question
-var typeface = new Array(); //GARAMOND
-typeface[0] = new Font(
-    ".garamond",
-    ".garamond",
-    ["Verdana", "Garamond", "Monotype"],
-    "Garamond"
-);
-console.log(typeface);
-
-typeface[1] = new Font(  //HELVETICA
-    ".helvetica",
-    ".helvetica",
-    ["Baskerville", "Sans-serif", "Helvetica"],
-    "Helvetica"
-);
-console.log(typeface);
-
-typeface[2] = new Font( //TIMES
-    ".times",
-    ".times",
-    ["Goudy Old Style", "Serifa", "Times"],
-    "Times"
-);
-console.log(typeface);
-
-typeface[3] = new Font( //VERDANA
-    ".verdana",
-    ".verdana",
-    ["Verdana", "Verdy", "Script"],
-    "Verdana"
-);
-console.log(typeface);
-
-typeface[4] = new Font( //ARIAL
-    ".arial",
-    ".arial",
-    ["Comic Sans", "Trade Gothic", "Arial"],
-    "Arial"
-);
-console.log(typeface);
```

Comment:
Good stuff! Alright, so now I've tweaked my code up a bit... I've removed the "sections" in html and the styling for it in CSS. So I tried to stylize it in javascript!

![screen shot 2015-07-01 at 3 46 57 pm](https://cloud.githubusercontent.com/assets/12088786/8463467/75d3ce10-2008-11e5-9e48-485b575ae10e.png)

Am I getting closer?